src/components/ToggleButton.css

```diff
@@ -1,35 +1,52 @@
 .onoffswitch {
-    position: relative; width: 40px;
-    margin-top: 5px;
+    position: relative; width: 43px;
     -webkit-user-select:none; -moz-user-select:none; -ms-user-select: none;
 }
 .onoffswitch-checkbox {
     display: none;
 }
 .onoffswitch-label {
     display: block; overflow: hidden; cursor: pointer;
-    height: 13px; padding: 0; line-height: 13px;
-    border: 0px solid #FFFFFF; border-radius: 20px;
-    background-color: #42A5F5;
+    border: 2px solid #999999; border-radius: 10px;
 }
-.onoffswitch-label:before {
+.onoffswitch-inner {
+    display: block; width: 200%; margin-left: -100%;
+    transition: margin 0.3s ease-in 0s;
+}
+.onoffswitch-inner:before, .onoffswitch-inner:after {
+    display: block; float: left; width: 50%; height: 17px; padding: 0; line-height: 17px;
+    font-size: 14px; color: white; font-family: Trebuchet, Arial, sans-serif; font-weight: bold;
+    box-sizing: border-box;
+}
+.onoffswitch-inner:before {
     content: "";
-    display: block; width: 20px; margin: -3.5px;
-    background: #2196F6;
-    position: absolute; top: 0; bottom: 0;
-    right: 23px;
-    border-radius: 20px;
+    padding-left: 10px;
+    background-color: #21304A; color: #5A9EEE;
 }
-.onoffswitch-checkbox:checked + .onoffswitch-label {
-    background-color: #42A5F5;
+.onoffswitch-inner:after {
+    content: "";
+    padding-right: 10px;
+    background-color: #21304A; color: #FFFFFF;
```

Comment:
what are the rules here for css rules in one line?

I see different ways off handling these.

Colors in one line? 
Layout in one line?